deploy/AzureAppServices/Datadog.Trace.AzureAppServices.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+ 
+  <PropertyGroup>
+    <!-- TODO: Get rid of this if possible -->
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+
+  <!-- This is a content only project, it exists only to auto-update so we see the entire site extensions package automatically -->
```

Comment:
> it exists only to auto-update

What is auto-updated using this project?

Reply:
the viewed content in visual studio basically. It was just for my ability to see what was happening from a "single pane of glass"

Reply:
Can you mention in the comment that this is all about the VS UI? Thanks.